lib/Runtime/Debug/TTEventLog.cpp

```diff
@@ -429,7 +429,13 @@ namespace TTD
             this->SetSnapshotOrInflateInProgress(true);
             this->PushMode(TTDMode::ExcludedExecutionTTAction);
 
-            snap = this->DoSnapshotExtract_Helper(0.0);
+            JsUtil::BaseHashSet<Js::FunctionBody*, HeapAllocator> liveTopLevelBodies(&HeapAllocator::Instance);
```

Comment:
are the `JsUtil::` classes chakra-internal collection classes?  Or are they types exposed through the JS runtime to users' processes?

Reply:
The are chakra-internal collections.